index.html

```diff
@@ -56,6 +56,7 @@ <h2 class="subtitle">Apache Kafka is a powerful, scalable, fault-tolerant distri
             <li><a href="creating-first-apache-kafka-consumer-application/kafka.html">Your first Kafka consumer application</a></li>
             <li><a href="kafka-producer-callback-application/kafka.html">Callbacks</a></li>
             <li><a href="produce-consume-lang/scala.html">Clients in multiple languages</a></li>
+            <li><a href="error-handling/kstreams.html">Error handling</a></li>
```

Comment:
@bbejeck note that there is an existing use case for "Handle deserialization errors" (https://kafka-tutorials.confluent.io/handling-deserialization-errors/ksql.html).  It would be good to think about how to name each of them so that they are clear & distinct and won't be confused with eachother.  (e.g. right now, "Error handling" sounds like a superset that could include "Handle deserialization errors" -- maybe this isn't ideal?)